src/routes/HomePage/HomePage.js

```diff
@@ -44,12 +62,33 @@ export default function HomePage() {
     );
   }
 
+  const linkToPets = () => {
+    navigate(`/pets`);
+  };
+
   const renderFeaturedPets = () => {
-    return (
-      <div className="container text-center m-3">
-        <h2>Add featured pets section in PM-43</h2>
-      </div>
-    );
+    if (pets) {
+      return (
+        <div className="container text-center m-3">
+          <h1 className="title text-info font-weight-bold">Meet Our New Pets!</h1>
```

Comment:
Do you think this should be in all caps to match the text in the image?

Reply:
fixed